src/components/auth/RegistrationForm.js

```diff
@@ -115,12 +115,14 @@ class RegistrationForm extends Component {
 		const {
 			college,
 			collegeChanged,
+			major,
 			majorChanged
 		} = this.props;
 
-		const p1 = college && college !== "" && college != "Do not wish to disclose"
+		const p1 = colleges && college !== ""
```

Comment:
Since "" is false, this can be changed to `colleges && college`, unless explicitly needed or ends up being confusing.